runtime-api/src/lib.rs

```diff
@@ -0,0 +1,21 @@
+// KILT Blockchain – https://botlabs.org
+// Copyright (C) 2019-2022 BOTLabs GmbH
+
+// The KILT Blockchain is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// The KILT Blockchain is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+// If you feel like getting in touch with us, you can do so at info@botlabs.org
+#![cfg_attr(not(feature = "std"), no_std)]
+
+pub mod did;
```

Comment:
I am not sure making each runtime API a module rather than a crate on its own is better. Having separate crates does not force us to compile everything every time if only some of the APIs are used (for instance mashnet-node does not use staking stuff). I would rather make each API a different crate. What do you guys think?

Reply:
Something like the `rpc` folder, where each subfolder is a different crate. I would see the `runtime-api` folder be structured in the same way.

Reply:
That seems to make sense.